packages/kbn-securitysolution-es-utils/kibana.jsonc

```diff
@@ -1,5 +1,9 @@
 {
   "type": "shared-common",
   "id": "@kbn/securitysolution-es-utils",
-  "owner": "@elastic/security-detection-engine"
-}
+  "owner": [
+    "@elastic/security-detection-engine"
+  ],
+  "group": "platform",
+  "visibility": "shared"
```

Comment:
I think something that's called `securitysolution-es-utils` should not be shared between the platform and all the solutions. But I do see that it's currently used from the `rule_registry` which is a part of the Alerting Framework.

My question is: do the new categorization properties added in this PR represent the current or desired state of things?

Reply:
They represent the current state. For this particular case, there's an [ongoing discussion](https://elastic.slack.com/archives/C6E3MTCD7/p1731405049134829).
The goal is to make some of the `@kbn/securitysolution-*` packages part of "platform" group.
The `securitysolution-` prefix will be missleading, so it should be stripped off as a follow-up task.

Reply:
Thanks for the clarification and sharing this link - it's a very helpful thread to follow 👍